scripts/localVsDissecLearning.js

```diff
@@ -0,0 +1,238 @@
+const fs = require('fs')
+const { v4: uuid } = require('uuid')
+const {
+  default: CozyClient,
+  createClientInteractive,
+  Q
+} = require('cozy-client')
+
+const { BANK_DOCTYPE } = require('../src/doctypes/bank')
+const { JOBS_DOCTYPE } = require('../src/doctypes/jobs')
+const { NODES_DOCTYPE } = require('../src/doctypes/nodes')
+const { PERFORMANCES_DOCTYPE } = require('../src/doctypes/performances')
+const dissecConfig = require('../dissec.config.json')
+
+const aggregationNodes = require('../assets/webhooks.json')
+const { exit } = require('process')
+
+/**
+ * This script measures performances of DISSEC vs local learning.
```

Comment:
I think it would be worth it to better explain what this script does, to avoid having to understand the code to know about it.